src/http-auth-interceptor.js

```diff
@@ -10,7 +10,18 @@
 
   angular.module('http-auth-interceptor', ['http-auth-interceptor-buffer'])
 
-  .factory('authService', ['$rootScope','httpBuffer', function($rootScope, httpBuffer) {
+  	/**
```

Comment:
This formatting is a mess. You have to adapt to the formatting of this file or it'll never be merged.